internal/pkg/utils/etcdhelper/assert.go

```diff
@@ -12,15 +13,68 @@ import (
 	etcd "go.etcd.io/etcd/client/v3"
 )
 
-// AssertKVs dumps all KVs from an etcd database and compares them with the expected string.
+// AssertKVsString dumps all KVs from an etcd database and compares them with the expected string.
```

Comment:
Vylepsil som pracu s `etcd` stavom v testoch.

Uz sa to neporovnava cele ako jeden velky string, ale kazdy objekt sa porovnava samostatne.
Jednoduchsie sa tak hlada chyba.

Prvym krokom bolo rozparsovat string na samostatne objekty.

Preto je tu `AssertKVsString`, a nova funkcia `AssertKVs`, ktora pouziva uz `[]KV`.